### PR TITLE
Replacement of Baselines in Client Devices Design

### PR DESCRIPTION
**Pull request by SamBH**
Replacing Windows 10 security baselines section (including Defender ATP and Edge) with Windows 10 Hardening and Microsoft Edge hardening sections.  This eliminates the blueprint's dependency on the Microsoft-provided baselines (which are subject to change) and increases compliance with the ACSC Hardening Guide for Windows 10.

### CODE REVIEW

File: blueprint/client-devices.md [hunk 1]
```diff
@@ -895,89 +895,371 @@ Microsoft Defender Smart Screen Configuration | Enabled and configured | Meets A
...
+  * Run these programs at user logon: Disabled
+* Microsoft accounts
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * Block all consumer Microsoft account user authentication: Enabled
+  * Prevent the usage of OneDrive for file storage: Enabled
+  * Accounts: Block Microsoft accounts: Users can’t add or log on with Microsoft accounts
+* MSS settings
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * IP source routing protection level: Enabled
+  * IP source routing protection level: Enabled
```
Comment (zedzedzed) [inline blueprint/client-devices.md:1109]: These appear twice and were there in the original formatting. Is there a missing setting or incorrect double paste?

**Comment by SamBH** [comment]
Probably a double paste I'd say sorry.

Cheers,

Sam Brazier-Hollins
Managing Consultant
www.oobe.com.au<https://aus01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.oobe.com.au%2F&data=04%7C01%7C%7C28e25c8a2f4746eb508b08d937803450%7C05f72b90815c4cb98b4096c29b862189%7C0%7C0%7C637601846075445925%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=iAKRFEoJysGq5ANGoKCk5IbhY2VqDN1khNlN9q6fj28%3D&reserved=0>
t: +61 2 6162 1447 | m: +61 409 903 349

***@***.***

File: blueprint/client-devices.md [hunk 2]
```diff
@@ -895,89 +895,371 @@ Microsoft Defender Smart Screen Configuration | Enabled and configured | Meets A
...
+    * Deny write access: Enabled
+  * WPD Devices:
+    * Deny read access: Disabled
+    * Deny write access: Enabled
+* File and print sharing
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * Prevent the computer from joining a homegroup: Enabled
+  * Prevent users from sharing files within their profile: Enabled
+* Hard drive encryption
+  * Justification: To align with the ACSC Windows 10 hardening guidance, with the exception of pre-boot PIN for useability.
```
Comment (zedzedzed) [inline blueprint/client-devices.md:1075]: What was the `pre-boot` option set to in the superseded baseline?

**Comment by SamBH** [comment]
I don't believe it was explicitly documented one way or another.

The agencies we have deployed the blueprint to tend to stay away from the pre-boot PIN, and therefore align with the recommended update.

Cheers,

Sam Brazier-Hollins
Managing Consultant
www.oobe.com.au<https://aus01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.oobe.com.au%2F&data=04%7C01%7C%7C28e25c8a2f4746eb508b08d937803450%7C05f72b90815c4cb98b4096c29b862189%7C0%7C0%7C637601846075445925%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=iAKRFEoJysGq5ANGoKCk5IbhY2VqDN1khNlN9q6fj28%3D&reserved=0>
t: +61 2 6162 1447 | m: +61 409 903 349

***@***.***

**Comment by zedzedzed** [comment]
> The agencies we have deployed the blueprint to tend to stay away from the pre-boot PIN

All fine for those agencies as I assume they would have been informed and made a concious educated decision to deviate from ACSC guidance that still fit within their risk profile. We also suspect some agencies may wish to keep it enabled so hard to please both.

In this case, I believe we should adopt ACSC's recommendation as we're creating a set of defaults, and those set of defaults should be as close to ACSCs guidance as possible. Agencies can choose to adopt as is or customise to their business needs, but they'd be customising with their eyes wide open.

**Review by zedzedzed — APPROVED** [review]

**Pull request merged**